modules/oc-mirror-workflows-fully-disconnected-v2.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-workflows-fully-disconnected-v2_{context}"]
+= Mirroring an image set in a fully disconnected environment
+
+In a fully disconnected environment, the mirror registry that the Openshift cluster can access is not reachable from the public internet. 
+
+Following are the steps of the workflow:
+
+. Mirror to disk: preparing an archive containing the image set that needs to be mirrored. This step requires internet access.
+
+. Manual step: the user transfers the archive to the network of the mirror registry, that is, disconnected registry.
+
+. Disk to mirror: from the environment that has access to the mirror registry, execute oc-mirror to mirror the image set from the archive to the target disconnected registry.
+
+[id="mirror-to-disk-v2"]
+== Mirroring from mirror to disk
+
+You can use the oc-mirror plugin to generate an image set and save the contents to disk. The generated image set can then be transferred to the disconnected environment and mirrored to the target registry.
+
+oc-mirror plugin pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk, that is, local directory.
+
+.Procedure
+
+. Copy the container images from the tar archive to a container registry.
```

Comment:
First add steps to mirror from Mirror To Disk then add the command on how to do the same.. Lineno.28 does not make sense with out actually mirroring first.